llm/llm.go

```diff
@@ -50,92 +50,88 @@ func New(model string, adapters, projectors []string, opts api.Options) (LLM, er
 	}
 
 	vram, _ := gpu.CheckVRAM()
-	size := ggml.Size
+	info := gpu.GetGPUInfo()
+
+	mem := int64(396 * 1024 * 1024)
+	for _, projector := range projectors {
+		mem += projectorMemoryRequirements(projector)
+	}
 
-	// fp16 k,v matrices require = n_ctx * n_layer * n_embd / n_head * n_head_kv * 2 bytes each * 2 key and value
-	kv := 2 * 2 * int64(opts.NumCtx) * int64(ggml.KV().BlockCount()) * int64(ggml.KV().EmbeddingLength()) * int64(ggml.KV().HeadCountKV()) / int64(ggml.KV().HeadCount())
+	// fp16 k,v = (1 (k) + 1 (v)) * sizeof(float16) * n_ctx * n_layer * n_embd / n_head * n_head_kv
```

Comment:
Can you explain the reasoning for changes in this calculation?

Reply:
There's no functional change here. I'm rearrange the terms to match the comment